src/main/webapp/index.html

```diff
@@ -4,9 +4,102 @@
     <meta charset="UTF-8">
     <title>Habit Stacker</title>
     <link rel="stylesheet" href="style.css">
+    <!-- Bootstrap implementation-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
   </head>
   <body onload="loadHabits()">
     <main>
+        <div id="LoginScreen">
+            <button class="w-100 btn btn-lg btn-primary" id="login">Login with Google</button>
+          </div>
+      
+          <div id="dashboard">
+            <div id="userDetails"></div>
+            <button id="logout">Logout</button>
+          </div>
+    
+          <!-- Boostrap and Popper (Bootstrap for dropdowns and extra stuff, added just incase we needed it)-->
+          <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+          <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.5/dist/umd/popper.min.js" integrity="sha384-Xe+8cL9oJa6tN/veChSP7q+mnSPaj5Bcu9mPX5F5xIGE0DVittaqT5lorf0EI7Vk" crossorigin="anonymous"></script>
+          <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.min.js" integrity="sha384-kjU+l4N0Yf4ZOJErLsIcvOU2qSb74wXpOhqTvwVx3OElZRweTnQ6d31fXEoRD1Jy" crossorigin="anonymous"></script>
+    
+          <!-- The core Firebase JS SDK is always required and must be listed first -->
+          <script src="https://www.gstatic.com/firebasejs/8.0.1/firebase-app.js"></script>
+      
+          <script src="https://www.gstatic.com/firebasejs/8.0.0/firebase-auth.js"></script>
+      
+          <!-- TODO: Add SDKs for Firebase products that you want to use
+           https://firebase.google.com/docs/web/setup#available-libraries -->
+      
+          <script>
+            // Web app's Firebase configuration
+            var firebaseConfig = {
+              apiKey: "AIzaSyB3iQ3tMOFzjAgmRWN5JxeQFLxrBVylq7E",
```

Comment:
Before submitting I want you to look into this: https://stackoverflow.com/questions/44342276/how-to-push-code-to-github-hiding-the-api-keys